src/website/src/pages/eu/getting-started/index.md

```diff
@@ -62,39 +64,44 @@ New components are continuously being added to the library. The team is also con
 </html>
 ```
 
-First:
-
-<ul>
-  <li>
-    Replace <code>[ECL_VERSION]</code> by{' '}
-    <code>v{process.env.ECL_VERSION}</code> to use the latest version
-  </li>
-  <li>
-    For more security, use these SRI hashes:
-    <ul>
-      <li>
-        for ecl-ec-preset-website.css:{' '}
-        <code>{process.env.ECL_EC_PRESET_WEBSITE_CSS}</code>
-      </li>
-      <li>
-        for ecl-ec-preset-website-print.css:{' '}
-        <code>{process.env.ECL_EC_PRESET_WEBSITE_PRINT_CSS}</code>
-      </li>
-      <li>
-        for ecl-ec-preset-website.js:{' '}
-        <code>{process.env.ECL_EC_PRESET_WEBSITE_JS}</code>
-      </li>
-    </ul>
-  </li>
-</ul>
-
-Then:
-
-- copy the markup from the template you want and paste it in the `<body>`.
+### Update file path and name
+
+- If you wish to use ECL EC instead of ECL EU, you can find information here <Link to="/ec/getting-started/" label="Getting started EC" />
+- Replace <code>[ECL_VERSION]</code> by <code>v{process.env.ECL_VERSION}</code> to use the latest version
+- For more security, use the SRI hashes corresponding to the files. They all can be found in the [release page](https://github.com/ec-europa/europa-component-library/releases), in a json file, but here are the most common ones:
+
+<details>
+  <summary>SRI hashes (click to expand)</summary>
+  
+  <ul>
+    <li>for `ecl-eu.css`: <code>{process.env.ECL_EU_CSS}</code></li>
```

Comment:
mmh...for how weird it is, we did not change the env variables names in webpack, meaning that the sri for `ecl-eu.css` is still `ECL_EU_PRESET_WEBSITE_CSS` how it was in v2.
I think we can fix this in this pull request, these are defined in both `webpack.config.prod.js` and `webpack.config.dev.js` (in the latter these are only "placeholders")